src/physics/convection_diffusion.h

```diff
@@ -62,7 +64,7 @@ class ConvectionDiffusion : public PhysicsBase <dim, nstate, real>
             hasConvection(convection), 
             hasDiffusion(diffusion)
     {
-        static_assert(nstate<=5, "Physics::ConvectionDiffusion() should be created with nstate<=5");
+        //static_assert(nstate<=5, "Physics::ConvectionDiffusion() should be created with nstate<=5");
```

Comment:
revert this change

Reply:
The compiler complains about it, I found it may be caused by the explicit instantiation for PhysicsBase with nstate = 6, which I need for RANS. Then, In DG it needs to create artificial dissipation with nstate = 6 and therefore I created the corresponding explicit instantiation. But then in LaplacianArtificialDissipation, the ConvectionDiffusion object is created, due to created explicit instantiation with nstate = 6, the corresponding ConvectionDiffusion then needs to be nstate = 6...
It seems there are so many ways to fix it, but for now, I just commented on this line, which may be the easiest.

Reply:
Ok, I'd just remove it if there's no reason it can't be created with nstate=6

Reply:
fixed